examples/todoapp/pubspec.yaml

```diff
@@ -0,0 +1,82 @@
+name: todoapp
+description: A new Flutter project.
+homepage: https://primeholding.com
+version: 1.0.0+1
+
+environment:
+  sdk: '>=3.4.3 <4.0.0'
+  flutter: '>=3.22.2 <4.0.0'
+
+dependencies: 
+  alice: ^0.4.1 
+  collection: ^1.18.0
+  copy_with_extension: ^5.0.4
+  cupertino_icons: ^1.0.6
+  dio: ^5.3.3
+  equatable: ^2.0.5
+  firebase_core: ^3.1.1
+  flutter:
+    sdk: flutter
+  flutter_localizations:
+    sdk: flutter
+  flutter_rx_bloc: ^6.1.1
+  flutter_secure_storage: ^9.0.0
+  go_router: ^14.2.0
+  golden_toolkit: ^0.15.0
+  http: ^1.1.0 # This package is used for the local server (if not used, can be removed)
+  jaguar_jwt: ^3.0.0 # This package is used for the local server (if not used, can be removed)
+  json_annotation: ^4.9.0
+  jwt_decoder: ^2.0.1
+  provider: ^6.0.5
+  retrofit: ^4.0.2
+  rx_bloc: ^5.0.1
+  rx_bloc_list: ^4.0.1
+  rxdart: ^0.27.7
+  shared_preferences: ^2.2.1
+  shelf: ^1.4.1 # This package is used for the local server (if not used, can be removed)
+  shelf_router: ^1.1.4 # This package is used for the local server (if not used, can be removed)
+  shelf_static: ^1.1.2 # This package is used for the local server (if not used, can be removed)
+  widget_toolkit: ^0.0.2  
+
+dev_dependencies:
+  build_runner: ^2.4.6
+  # flutter_driver:
+  #   sdk: flutter
+  clean_coverage: ^0.0.3
+  copy_with_extension_gen: ^5.0.4
+  flutter_lints: any
+  flutter_test:
+    sdk: flutter
+  go_router_builder: ^2.3.2
+  json_serializable: ^6.7.1
+  mockito: ^5.4.2 
+  r_flutter: 0.9.0
+  retrofit_generator: ^8.0.4
+  rx_bloc_generator: ^7.2.2
+  rx_bloc_test: ^4.0.0
+  test: any 
+
+flutter:
+  uses-material-design: true
+  assets:
+    - assets/images/
+    - assets/fonts/
+  fonts:
+    - family: WorkSans
+      fonts:
+        - asset: assets/fonts/WorkSans/WorkSans-Light.ttf
+          weight: 300
+        - asset: assets/fonts/WorkSans/WorkSans-Regular.ttf
+          weight: 400
+        - asset: assets/fonts/WorkSans/WorkSans-Medium.ttf
+          weight: 500
+
+r_flutter:
+  intl: lib/l10n/arb/en.arb
+  intl_features:
+    - name: error
+    - name: feature_deep_link
```

Comment:
Please remove all configurations related to not-used features

Reply:
Done